wallet/how-to/send-transactions/send-batch-transactions.md

```diff
@@ -0,0 +1,204 @@
+
+See the [MetaMask Delegation Toolkit documentation](https://docs.gator.metamask.io/) for more information about delegator accounts and their capabilities.
+:::
+
+The key benefits of atomic batch transactions include:
+
+- **Fewer clicks and less friction** - Users only need to review and approve a single wallet confirmation, instead of multiple confirmations.
+  For example, users can confirm a spending cap and swap in one step instead of two.
+- **Faster completion times** - Only a single atomic transaction is confirmed onchain, instead of multiple individual transactions.
+- **Reduced gas fees** - When multiple transactions are executed atomically, users only need to pay a single gas fee.
+
+You can send batch transactions using [third-party libraries](#use-third-party-libraries)
+or [directly in your dapp](#send-batch-transactions-1).
+
+## Use third-party libraries
+
+You can send batch transactions using the following third-party libraries that support EIP-5792:
+
+- [Wagmi](https://wagmi.sh/react/api/hooks/useCapabilities)
+- [Viem](https://viem.sh/experimental/eip5792/client)
+- [thirdweb](https://portal.thirdweb.com/references/typescript/v5/hooks#eip5792)
+
+## Send batch transactions
+
+### 1. Query whether atomic batch is supported
+
+Use `wallet_getCapabilities` to query whether MetaMask supports atomic batch transactions for a specific address and specific chain IDs.
+For example:
+
+```js title="index.js"
+const capabilities = await provider // Or window.ethereum if you don't support EIP-6963.
+  .request({
+    "method": "wallet_getCapabilities",
+    "params": [
+      "0xd46e8dd67c5d32be8058bb8eb970870f07244567", // The user's wallet address.
+      ["0x1", "0xaa36a7"] // (Optional) A list of chain IDs to query for.
+    ],
+  });
+```
+
+This method returns the status of the `atomic` capability for each chain ID.
+For example:
+
+```json
+{
+  "0x1": {
+    "atomic": {
+      "status": "ready"
+    }
+  },
+  "0xaa36a7": {
+    "atomic": {
+      "status": "supported"
+    }
+  }
+}
+```
+
+The `atomic` capability can have a `status` of `supported` or `ready`:
+
+- `supported` means MetaMask supports atomic batch transactions for the account and chain ID.
+- `ready` means MetaMask will prompt the user to upgrade their account to a MetaMask delegator account.
+  If the user approves, the `status` will upgrade to `supported`.
+
+If the `atomic` capability is not `supported` or `ready` for a specified chain ID, MetaMask will not return anything for that chain ID.
+If you don't specify any chain IDs in `wallet_getCapabilities`, MetaMask will return all chains in the wallet where the `atomic` capability is `supported` or `ready`.
+
+:::info Supported networks
+MetaMask currently supports atomic batch transactions on Ethereum Mainnet, Ethereum Sepolia, and Gnosis Mainnet.
```

Comment:
We might add `BNB Smart Chain` and `Gnosis Chiado` (Gnosis testnet) as well... we will enable those two ahead of (or with the mainnet pectra upgrade)